string color_str
---
PlayerState player